web/elm/tests/DashboardTests.elm

```diff
@@ -866,6 +866,39 @@ all =
                     |> Common.queryView
                     |> Query.find [ class "dashboard-team-name-wrapper" ]
                     |> Query.has [ style "letter-spacing" ".2em" ]
+        , describe "clicking on team name"
+            [ test
+                "filters by team when there are pipelines"
+              <|
```

Comment:
Hey @robwhitby, thanks for this PR! Do you mind doing a `yarn format` to make it easier on eyes? Thanks!

Reply:
yep, no problem